components/Layout/Footer/Footer.tsx

```diff
@@ -1,20 +1,32 @@
+import { ConditionalFeature } from '../../../lib/feature-flags/feature-flags';
+import { useAuth } from '../../UserContext/UserContext';
 import s from './Footer.module.scss';
 
-const Footer = (): React.ReactElement => (
-  <footer className={`${s.footer}`} role="contentinfo">
-    <div className="govuk-width-container">
-      <nav className={s.links}>
-        <a href="https://sites.google.com/hackney.gov.uk/moderntoolsforsocialcare/home">
-          Roadmap
-        </a>
-        <a href="https://docs.google.com/forms/d/e/1FAIpQLScILbPD1ioKHzp1D3HN4_DKaxV2tpWLMu8upSSqNgSPCo85cg/viewform">
-          Give feedback
-        </a>
-      </nav>
+const Footer = (): React.ReactElement => {
+  const { user } = useAuth();
 
-      <div className={s.meta}>Built and maintained by HackIT.</div>
-    </div>
-  </footer>
-);
+  return (
+    <footer className={`${s.footer}`} role="contentinfo">
+      <div className="govuk-width-container">
+        <nav className={s.links}>
+          <a href="https://sites.google.com/hackney.gov.uk/moderntoolsforsocialcare/home">
+            Roadmap
+          </a>
+          <a href="https://docs.google.com/forms/d/e/1FAIpQLScILbPD1ioKHzp1D3HN4_DKaxV2tpWLMu8upSSqNgSPCo85cg/viewform">
+            Give feedback
+          </a>
+        </nav>
+
+        <div className={s.meta}>Built and maintained by HackIT.</div>
+
+        <ConditionalFeature name="feature-flags-implementation-proof">
+          {user?.hasAdminPermissions && (
```

Comment:
Is there a difference between having `dev permissions` and having `admin permissions`?
What does `hasAdminPermissions` check for?

Reply:
Looks like it checks for either `admin` or `dev` permissions under the hood:

```
hasAdminPermissions:
      groups.includes(AUTHORISED_ADMIN_GROUP) ||
      groups.includes(AUTHORISED_DEV_GROUP),
```